frontend/src/pages/auth/Auth.styled.tsx

```diff
@@ -1,5 +1,5 @@
 import styled from "styled-components";
-import { FONT_SIZES } from "constants/styleConstants";
+import { FONT_SIZES } from "assets/styles/theme";
```

Comment:
There isn't consistency in notation of font-size. For example in Menu.styled.ts file, each value isn't as a const but as normal value.

Reply:
fixed